src/python/zquantum/core/distance_measures.py

```diff
@@ -0,0 +1,108 @@
+### The following code is derived from: https://github.com/GiggleLiu/QuantumCircuitBornMachine/blob/master/LICENSE
+### The original resource is under MIT license, which is pasted below for convenience >>>
+# MIT License
+# Copyright (c) 2017 Leo
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import numpy as np
+
+
+def compute_rbf_kernel(x_i, y_j, sigma):
+    """ Compute the gaussian (RBF) kernel matrix K, with K_ij = exp(-gamma |x_i - y_j|^2) and gamma = 1/(2*sigma).
+
+        Args:
+            x_i (np.array): Samples A (integers).
+            y_j (np.array): Samples B (integers).
+            sigma (float): The bandwidth of the gaussian kernel.
+
+        Returns:
+            np.ndarray: The gaussian kernel matrix.
+    """
+    exponent = np.abs(x_i[:, None] - y_j[None, :]) ** 2
+    try:
+        gamma = 1.0 / (2 * sigma)
+    except ZeroDivisionError as error:
+        print("Handling run-time error:", error)
+        raise
+    K = np.exp(-gamma * exponent)
+    return K
+
+
+def compute_multi_rbf_kernel(x_i, y_j, sigmas):
+    """ Compute the multi-gaussian (RBF) kernel matrix K, with K_ij = 1/c Sum_n[exp(-gamma_n |x_i - y_j|^2)] and gamma = 1/(2*sigma).
```

Comment:
I don't see `c` anywhere in the code 🤔

Reply:
`c` is the the normalization factor `len(sigmas)`.
The index `n` runs from 1 to `c`. 
Shall we clarify it in the docstring?

Reply:
I see. Yes, I think it should be clarified in the docs :)

Reply:
Addressed [here](https://github.com/zapatacomputing/z-quantum-core/pull/63/commits/1c8260e429fa744045ec72ed5aef29dd3f4ab3ff)